docker/Dockerfile

```diff
@@ -8,10 +8,9 @@ USER root
 SHELL [ "/bin/bash", "-exo", "pipefail", "-c" ]
 
 # Install some linux packages
-# awscli requires unzip, less, groff and mandoc
 # hadolint ignore=DL3008
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y git jq unzip less groff mandoc postgresql && \
+    apt-get install --no-install-recommends -y git jq postgresql && \
```

Comment:
Oh yeah I meant jq. I didn't realize they were different things.  I'll try removing `jq` and see if things still work. My guess was it was related to the AWS CLI, but I wasn't sure since it wasn't mentioned in the comment with the other dependencies.